src/views/pages/hardware/management.ejs

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <% include ../../partials/head.ejs %>
+  <title>GUH18 Hub - Dashboard</title>
+  <link rel="stylesheet" href="/css/hardwareManagement.css">
+  <link rel="stylesheet" href="/css/hardware.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <% include ../../partials/navbar.ejs %>
+    <div class="main-panel">
+      <div class="content">
+        <!-- Confirmation Modal -->
+        <div class="modal fade" id="confirmModal" tabindex="-1" role="dialog" aria-labelledby="confirmModalLabel" aria-hidden="true">
+            <div class="modal-dialog" role="document">
+              <div class="modal-content">
+                <div class="modal-header">
+                  <h5 class="modal-title" id="confirmModalLabel">Do you really want to delete this item?</h5>
+                </div>
+                <div class="modal-body">
+                  <div class="imgContainer">
+                    <img src="" alt="" id="confirmModalImg" class="modalImg">
+                  </div>
+                  <div id="confirmModalBody">
+                  </div>
+                </div>
+                <div class="modal-footer">
+                  <button id="modal-confirm-button" type="button" class="btn btn-success" data-dismiss="modal">Confirm</button>
+                  <button type="button" class="btn btn-danger" data-dismiss="modal">Cancel</button>
+                </div>
+              </div>
+            </div>
+          </div>
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-md-4">
+                <div class="row">
+                    <div id="update-panel" class="card" style="display: none">
+                          <div class="card-header card-header-tabs card-header-warning">
+                            <h4 class="card-title">Update item</h4>
+                          </div>
+                          <div class="card-body">
+                            <div class="row">
+                              <img id="update-img-preview" class="img-container">
```

Comment:
I think we want the class to be imgContainer, since currently the img-container has the attribute max-width=100% so when editing a hardware item the image can overflow the container.

Reply:
Yeah I definitely need to cleanup the class names. 😂 
What do you mean by the image overflowing the container? It seems to work fine for me

Reply:
Sorry, my mistake. Refreshed my cache and it works fine now